src/components/img.ts

```diff
@@ -1,7 +1,22 @@
 import { Component } from '../component'
 import { h, strToHash } from '../core'
 
-export class Img extends Component {
+interface Props {
+  [key: string]: any
+  src: string
+  height?: number | string
+  width?: number | string
+  lazy?: boolean
+  placeholder?: any
```

Comment:
I wanted to implement placeholder type as the type such a `ReactNode | string`, but I cannot found the type that corresponds to `ReactNode` in nanojsx. Do you know some type that corresponds to `ReactNode` in nanojsx?

Reply:
No, I don't know yet. I guess `core.ts` and `component.ts` first need some proper type definitions.

Reply:
OK, I'll try it 👍

Reply:
I quickly tried something but it failed horribley 🤣
https://github.com/nanojsx/nano/tree/tmp

Reply:
Oh really? I felt relieved because the test was passed 😭  Sorry, I should have been check about my change not only with test but also real environment 🙇  (btw, It would be better there is the test for more real environment(compile, bundle, render, update) like e2e)

Reply:
When I implement `NanoElement` type, I noticed that `placeholder` should get not `Element` but Component(Class Compnent or FC).
`NanoElement` type is not needed right now, so I will implement the type later(when I need this type for better type for nanojsx) 🙏  thank you for trying!